### PR TITLE
Add pagination to event query

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/type/Pagination.java [hunk 1]
```diff
@@ -18,11 +18,15 @@
...
 
+import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Getter
 @Setter
+@NoArgsConstructor
+@AllArgsConstructor
```
Comment (wu-sheng) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/type/Pagination.java:29]: Who is using `AllArgsConstructor`?

File: oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/AlarmQuery.java [hunk 2]
```diff
@@ -88,7 +88,9 @@ public Alarms getAlarm(final Duration duration, final Scope scope, final String
         }
         long startSecondTB = 0;
         long endSecondTB = 0;
-        final EventQueryCondition.EventQueryConditionBuilder conditionPrototype = EventQueryCondition.builder().size(IEventQueryDAO.MAX_SIZE);
+        final EventQueryCondition.EventQueryConditionBuilder conditionPrototype =
+            EventQueryCondition.builder()
+                               .paging(new Pagination(1, IEventQueryDAO.MAX_SIZE, false));
```
Comment (kezhenxu94) [inline oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/AlarmQuery.java:93]: > Who is using AllArgsConstructor?

Here

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
> I can't see GraphQL submodule update.

e2e still pass because no event related test cases, right?

**Review by wu-sheng — CHANGES_REQUESTED** [review]
I can't see GraphQL submodule update.

**Review by wu-sheng — COMMENTED** [review]
Do we need e2e case update? I think this is a new condition for client, right?

**Comment by kezhenxu94** [comment]
> Do we need e2e case update? I think this is a new condition for client, right?

E2E client didn't set `size` parameter, it uses the default value of it so we don't need to update.

> Do we need e2e case update? I think this is a new condition for client, right?

Thanks for reminding, updated

**Review by wu-sheng — APPROVED** [review]
LGTM, once e2e passed.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**